spec/factories/cart.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :cart do
+    content { {1 => 3, 2 => 3, 3 => 3} }
```

Comment:
the keys are being stored as integers as far as I can tell, since the params[:item_id] that we create the cart item with is an integer